.ci/maven-prepare.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z $1 ]]; then
+  echo "latest version is not set"
+  exit 1
+fi
+
+TARGET_VERSION=$1
+echo TARGET_VERSION="$TARGET_VERSION"
+
+sudo apt-get install xmlstarlet
```

Comment:
Move this to action.
Script should be ready to run on local

Reply:
Changes Done